### PR TITLE
DOC Update changelog API changes to include reasons

### PR DESCRIPTION
**Pull request by GuySartorelli**
Created changes using https://github.com/GuySartorelli/silverstripe-deprecator, then modified where necessary to make it cleaner for human eyes and to fix any errors.

## Issue
- https://github.com/silverstripe/developer-docs/issues/203

### CODE REVIEW

File: en/04_Changelogs/5.0.0.md [hunk 1]
```diff
@@ -742,13 +749,13 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
 
 #### silverstripe/auditor
 
-- Removed deprecated method `SilverStripe\Auditor\AuditHook::bind_manipulation_capture()`
-- Removed deprecated method `SilverStripe\Auditor\AuditHook::onBeforeInit()`
+- Removed deprecated method `SilverStripe\Auditor\AuditHook::bind_manipulation_capture()` - use `ProxyDBExtension` with the `tractorcow/silverstripe-proxy-db` module instead
+- Removed deprecated method `SilverStripe\Auditor\AuditHook::onBeforeInit()` - use `tractorcow/silverstripe-proxy-db` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:753]: Is it possible to add more details what exactly use instead?

File: en/04_Changelogs/5.0.0.md [hunk 2]
```diff
@@ -760,31 +767,31 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Class `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_ArchiveLink` has been moved from `silverstripe/cms` to `silverstripe/versioned-admin` and renamed [`SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_ArchiveLink`](api:SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_ArchiveLink)
+- Class `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_LiveLink` has been moved from `silverstripe/cms` to `silverstripe/versioned-admin` and renamed [`SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_LiveLink`](api:SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_LiveLink)
+- Class `SilverStripe\CMS\Controllers\SilverStripeNavigatorItem_StageLink` has been moved from `silverstripe/cms` to `silverstripe/versioned-admin` and renamed [`SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_StageLink`](api:SilverStripe\VersionedAdmin\Navigator\SilverStripeNavigatorItem_StageLink)
+- Removed deprecated class `SilverStripe\CMS\Controllers\CMSPageHistoryController` - use `silverstripe/versioned-admin` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:784]: Could we provide fully qualified name of the class in silverstripe/versioned-admin? 
Like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 3]
```diff
@@ -760,31 +767,31 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated class `SilverStripe\CMS\Model\SiteTreeFileExtension` - use `FileLinkTracking` instead
+- Removed deprecated class `SilverStripe\CMS\Model\SiteTreeFileFormFactoryExtension` - use `UsedOnTable` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:786]: Here as well.

File: en/04_Changelogs/5.0.0.md [hunk 4]
```diff
@@ -848,19 +855,19 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
 
 #### silverstripe/comments
 
-- Removed deprecated class `SilverStripe\Comments\Admin\CommentsGridFieldAction`
-- Removed deprecated class `SilverStripe\Comments\Admin\CommentsGridFieldBulkAction\Handler`
-- Removed deprecated method `SilverStripe\Comments\Model\Comment::getParent()`
+- Removed deprecated class `SilverStripe\Comments\Admin\CommentsGridFieldAction` - use `CommentsGridFieldApproveAction` or `CommentsGridFieldSpamAction` instead
+- Removed deprecated class `SilverStripe\Comments\Admin\CommentsGridFieldBulkAction\Handler` - use concrete Spam or Approve handlers instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:859]: FQN here as well.
Like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 5]
```diff
@@ -900,228 +907,231 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated class `SilverStripe\Core\Startup\ErrorControlChainMiddleware` without equivalent functionality to replace it
+- Removed deprecated class `SilverStripe\Core\Startup\ErrorDirector` without equivalent functionality to replace it
+- Removed deprecated class `SilverStripe\Core\Startup\ParameterConfirmationToken` without equivalent functionality to replace it
+- Removed deprecated class `SilverStripe\Core\Startup\URLConfirmationToken` without equivalent functionality to replace it
+- Removed deprecated class `SilverStripe\Dev\CSVParser` - use `League\Csv\Reader` instead
+- Removed deprecated class `SilverStripe\Dev\InstallerTest` without equivalent functionality to replace it
+- Removed deprecated class `SilverStripe\Dev\SapphireInfo` without equivalent functionality to replace it
+- Removed deprecated class `SilverStripe\Dev\Tasks\MigrateFileTask` without equivalent functionality to replace it
+- Removed deprecated class `SilverStripe\Dev\TestSession_STResponseWrapper` without equivalent functionality to replace it
+- Removed deprecated class `SilverStripe\Forms\GridField\GridFieldVersionedState` - use `VersionedGridFieldState` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:943]: FQN in `silverstripe/versioned`.
Like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 6]
```diff
@@ -900,228 +907,231 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated class `SilverStripe\Forms\HTMLEditor\TinyMCEGZIPGenerator` without equivalent functionality to replace it
+- Removed deprecated class `SilverStripe\ORM\Connect\PDOConnector` without equivalent functionality to replace it
+- Removed deprecated class `SilverStripe\ORM\Connect\PDOQuery` without equivalent functionality to replace it
+- Removed deprecated class `SilverStripe\ORM\Connect\PDOStatementHandle` without equivalent functionality to replace it
+- Removed deprecated class `SilverStripe\ORM\Map_Iterator` without equivalent functionality to replace it
+- Removed deprecated class `SilverStripe\View\Embed\EmbedResource` - use `EmbedContainer` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:949]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 7]
```diff
@@ -900,228 +907,231 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated class `SilverStripe\View\Parsers\Diff` - use [`SilverStripe\View\Parsers\HtmlDiff`](api:SilverStripe\View\Parsers\HtmlDiff) instead
+- Removed deprecated class `SilverStripe\View\Parsers\HTML4Value` - use [`SilverStripe\View\Parsers\HTMLValue`](api:SilverStripe\View\Parsers\HTMLValue) instead
+- Removed deprecated interface `SilverStripe\Control\Email\Mailer` - use `symfony/mailer` instead
+- Removed deprecated interface `SilverStripe\Control\RequestFilter` - use `HTTPMiddleware` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:953]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 8]
```diff
@@ -900,228 +907,231 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated method `SilverStripe\Control\Email\Email::hasPlainPart()` without equivalent functionality to replace it
+- Removed deprecated method `SilverStripe\Control\Email\Email::invalidateBody()` - use `html()` instead
+- Removed deprecated method `SilverStripe\Control\Email\Email::IsEmail()` - use `getData()` instead
+- Removed deprecated method `SilverStripe\Control\Email\Email::mergeConfiguredEmails()` without equivalent functionality to replace it
+- Removed deprecated method `SilverStripe\Control\Email\Email::render()` without equivalent functionality to replace it
+- Removed deprecated method `SilverStripe\Control\Email\Email::setFailedRecipients()` without equivalent functionality to replace it
+- Removed deprecated method `SilverStripe\Control\Email\Email::setSwiftMessage()` without equivalent functionality to replace it
+- Removed deprecated method `SilverStripe\Control\HTTP::add_cache_headers()` - headers are added automatically by `HTTPCacheControlMiddleware` instead.
+- Removed deprecated method `SilverStripe\Control\HTTP::augmentState()` - use `HTTPCacheControlMiddleware` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:971]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 9]
```diff
@@ -900,228 +907,231 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated method `SilverStripe\Control\HTTP::get_cache_age()` without equivalent functionality to replace it
+- Removed deprecated method `SilverStripe\Control\HTTP::gmt_date()` - use native php function [`gmdate()`](https://www.php.net/manual/en/function.gmdate.php) instead
+- Removed deprecated method `SilverStripe\Control\HTTP::register_etag()` - use `ChangeDetectionMiddleware` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:974]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 10]
```diff
@@ -900,228 +907,231 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated method `SilverStripe\Dev\Deprecation::set_enabled()` - use `enable()` instead
+- Removed deprecated method `SilverStripe\Dev\FunctionalTest::get_use_draft_site()` - use `?stage=Stage` in your request's querystring instead
+- Removed deprecated method `SilverStripe\Dev\FunctionalTest::useDraftSite()` - use `?stage=Stage` in your request's querystring instead
+- Removed deprecated method `SilverStripe\Dev\SapphireTest::assertDOSAllMatch()` - use `assertListAllMatch()` instead
+- Removed deprecated method `SilverStripe\Dev\SapphireTest::assertDOSContains()` - use `assertListContains()` instead
+- Removed deprecated method `SilverStripe\Dev\SapphireTest::assertDOSEquals()` - use `assertListEquals()` instead
+- Removed deprecated method `SilverStripe\Dev\SapphireTest::assertNotDOSContains()` - use `assertListNotContains()` instead
+- Removed deprecated method `SilverStripe\Dev\SapphireTest::getFixtureFactory()` - use `FixtureTestState` instead
+- Removed deprecated method `SilverStripe\Dev\SapphireTest::loadFixture()` - use `FixtureTestState` instead
+- Removed deprecated method `SilverStripe\Dev\SapphireTest::setFixtureFactory()` - use `FixtureTestState` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:1028]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 11]
```diff
@@ -900,228 +907,231 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated method `SilverStripe\View\ViewableData::ThemeDir()` - use `ModuleResourceLoader::resourcePath()` or `ModuleResourceLoader::resourceURL()` instead
+- Removed deprecated config `SilverStripe\Control\Director.alternate_public_dir` without equivalent functionality to replace it
 - Removed deprecated config `SilverStripe\Control\HTTP.cache_ajax_requests`
-- Removed deprecated config `SilverStripe\Control\HTTP.cache_control`
-- Removed deprecated config `SilverStripe\Control\HTTP.disable_http_cache`
-- Removed deprecated config `SilverStripe\Control\HTTP.vary`
-- Removed deprecated config `SilverStripe\Forms\GridField\GridFieldFilterHeader.force_legacy`
+- Removed deprecated config `SilverStripe\Control\HTTP.cache_control` - use `HTTPCacheControlMiddleware` instead
+- Removed deprecated config `SilverStripe\Control\HTTP.disable_http_cache` - use `HTTPCacheControlMiddleware.defaultState.defaultForcingLevel` instead
+- Removed deprecated config `SilverStripe\Control\HTTP.vary` - use `HTTPCacheControlMiddleware` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:1101]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L1103

File: en/04_Changelogs/5.0.0.md [hunk 12]
```diff
@@ -900,228 +907,231 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated constant `SilverStripe\Core\Manifest\Module::CI_PHPUNIT_FIVE` without equivalent functionality to replace it
+- Removed deprecated constant `SilverStripe\Core\Manifest\Module::CI_PHPUNIT_NINE` without equivalent functionality to replace it
+- Removed deprecated constant `SilverStripe\Core\Manifest\Module::CI_UNKNOWN` without equivalent functionality to replace it
+- Removed deprecated constant `SilverStripe\Core\Manifest\Module::TRIM_CHARS` - use `Path::normalise()` instead
+- Removed deprecated property `SilverStripe\Control\Controller::$basicAuthEnabled` - add this controller's url to [`SilverStripe\Security\BasicAuthMiddleware.URLPatterns`](api:SilverStripe\Security\BasicAuthMiddleware.URLPatterns) injected property instead of setting false
+- Removed deprecated property `SilverStripe\Control\HTTP::$cache_age` - use `HTTPCacheControlMiddleware::singleton()->setMaxAge($age)` instead
+- Removed deprecated property `SilverStripe\Control\HTTP::$etag` - use `ChangeDetectionMiddleware` instead
+- Removed deprecated property `SilverStripe\Control\HTTP::$modification_date` - use `HTTPCacheControlMiddleware` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:1117]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 13]
```diff
@@ -1486,7 +1499,7 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
 
 #### silverstripe/hybridsessions
 
-- Removed deprecated class `SilverStripe\HybridSessions\Crypto\McryptCrypto`
+- Removed deprecated class `SilverStripe\HybridSessions\Crypto\McryptCrypto` - use `OpenSSLCrypto` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:1502]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 14]
```diff
@@ -1688,20 +1699,21 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
 - Changed parameter type in [`Symbiote\QueuedJobs\Services\QueuedJobHandler::write()`](api:Symbiote\QueuedJobs\Services\QueuedJobHandler::write()) for `$record` from `array` to `Monolog\LogRecord`
 
 #### tractorcow/silverstripe-fluent
 
-- Removed deprecated method `TractorCow\Fluent\Extension\FluentExtension::getLinkingMode()`
-- Removed deprecated method `TractorCow\Fluent\Extension\FluentExtension::LocaleLink()`
-
-<!--- Changes below this line will be automatically regenerated -->
+- Removed deprecated method `TractorCow\Fluent\Extension\FluentExtension::getLinkingMode()` - use `LocaleInformation()` instead
+- Removed deprecated method `TractorCow\Fluent\Extension\FluentExtension::LocaleLink()` - use `LocaleInformation` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:1717]:
```suggestion
- Removed deprecated method `TractorCow\Fluent\Extension\FluentExtension::LocaleLink()` - use `LocaleInformation()` instead
```

File: en/04_Changelogs/5.0.0.md [hunk 15]
```diff
@@ -1549,19 +1562,19 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated method `SilverStripe\LDAP\Extensions\LDAPMemberExtension::memberLoggedIn()` without equivalent functionality to replace it
 
 #### silverstripe/mfa
 
-- Removed deprecated method `SilverStripe\MFA\Report\EnabledMembers::formatDefaultMethodColumn()`
-- Removed deprecated method `SilverStripe\MFA\Report\EnabledMembers::formatMethodsColumn()`
-- Removed deprecated method `SilverStripe\MFA\Store\SessionStore::serialize()`
-- Removed deprecated method `SilverStripe\MFA\Store\SessionStore::unserialize()`
+- Removed deprecated method `SilverStripe\MFA\Report\EnabledMembers::formatDefaultMethodColumn()` - use `MemberExtension::getDefaultRegisteredMethodName()` instead
+- Removed deprecated method `SilverStripe\MFA\Report\EnabledMembers::formatMethodsColumn()` - use `MemberExtension::getRegisteredMethodNames()` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:1570]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 16]
```diff
@@ -900,228 +907,231 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated method `SilverStripe\ORM\DatabaseAdmin::updateLegacyClassNames()` - use `updateLegacyClassNameField()` instead
+- Removed deprecated method `SilverStripe\ORM\DataExtension::unload_extra_statics()` without equivalent functionality to replace it
+- Removed deprecated method `SilverStripe\ORM\DataList::getGenerator()` without equivalent functionality to replace it
+- Removed deprecated method `SilverStripe\ORM\DataObject::doValidate()` - use `validate()` instead
+- Removed deprecated method `SilverStripe\ORM\DataObject::duplicateManyManyRelations()` - use `duplicateRelations()` instead
+- Removed deprecated method `SilverStripe\ORM\DB::getConn()` - use `DB::get_conn()` instead
+- Removed deprecated method `SilverStripe\ORM\FieldType\DBClassName::clear_classname_cache()` - use `DBEnum::flushCache()` instead
+- Removed deprecated method `SilverStripe\ORM\ValidationResult::serialize()` - use `__serialize()` instead
+- Removed deprecated method `SilverStripe\ORM\ValidationResult::unserialize()` - use `__unserialize()` instead
+- Removed deprecated method `SilverStripe\Security\Member::checkPassword()` - use `Authenticator::checkPassword()` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:1064]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 17]
```diff
@@ -900,228 +907,231 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated method `SilverStripe\Security\Member::create_new_password()` without equivalent functionality to replace it
+- Removed deprecated method `SilverStripe\Security\Member::currentUser()` - use `Security::getCurrentUser()` instead
+- Removed deprecated method `SilverStripe\Security\Member::currentUserID()` - use `Security::getCurrentUser()` instead
+- Removed deprecated method `SilverStripe\Security\Member::default_admin()` - use `DefaultAdminService::findOrCreateDefaultAdmin()` instead
+- Removed deprecated method `SilverStripe\Security\Member::logged_in_session_exists()` - use `Security::getCurrentUser()` instead
+- Removed deprecated method `SilverStripe\Security\Member::logIn()` - use `Security::setCurrentUser()` or `IdentityStore::logIn()` instead
+- Removed deprecated method `SilverStripe\Security\Member::logOut()` - use `Security::setCurrentUser(null)` or an `IdentityStore` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:1071]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 18]
```diff
@@ -900,228 +907,231 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated method `SilverStripe\Security\Security::check_default_admin()` - use `DefaultAdminService::isDefaultAdminCredentials()` instead
+- Removed deprecated method `SilverStripe\Security\Security::clear_default_admin()` - use `DefaultAdminService::clearDefaultAdmin()` instead
+- Removed deprecated method `SilverStripe\Security\Security::default_admin_password()` - use `DefaultAdminService::getDefaultAdminPassword()` instead
+- Removed deprecated method `SilverStripe\Security\Security::default_admin_username()` - use `DefaultAdminService::getDefaultAdminUsername()` instead
+- Removed deprecated method `SilverStripe\Security\Security::findAnAdministrator()` - use `DefaultAdminService::findOrCreateDefaultAdmin()` instead
+- Removed deprecated method `SilverStripe\Security\Security::getLoginForms()` - use `delegateToMultipleHandlers()` instead
+- Removed deprecated method `SilverStripe\Security\Security::has_default_admin()` - use `DefaultAdminService::hasDefaultAdmin()` instead
+- Removed deprecated method `SilverStripe\Security\Security::setDefaultAdmin()` - use `DefaultAdminService::setDefaultAdmin($username, $password)` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:1087]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 19]
```diff
@@ -900,228 +907,231 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated method `SilverStripe\View\Requirements_Backend::getMinifier()` without equivalent functionality to replace it
+- Removed deprecated method `SilverStripe\View\Requirements_Backend::getMinifyCombinedFiles()` without equivalent functionality to replace it
+- Removed deprecated method `SilverStripe\View\Requirements_Backend::setMinifier()` without equivalent functionality to replace it
+- Removed deprecated method `SilverStripe\View\Requirements_Backend::setMinifyCombinedFiles()` without equivalent functionality to replace it
+- Removed deprecated method `SilverStripe\View\Shortcodes\EmbedShortcodeProvider::embedForTemplate()` - use `embeddableToHtml()` instead
+- Removed deprecated method `SilverStripe\View\SSViewer::set_theme()` - use `SSViewer::set_themes()` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:1093]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 20]
```diff
@@ -900,228 +907,231 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated method `SilverStripe\View\SSViewer_BasicIteratorSupport::First()` - use `IsFirst()` instead
+- Removed deprecated method `SilverStripe\View\SSViewer_BasicIteratorSupport::Last()` - use `IsLast()` instead
+- Removed deprecated method `SilverStripe\View\ViewableData::ThemeDir()` - use `ModuleResourceLoader::resourcePath()` or `ModuleResourceLoader::resourceURL()` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:1096]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 21]
```diff
@@ -900,228 +907,231 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated method `SilverStripe\ORM\Connect\Query::rewind()` - use `getIterator()` instead
+- Removed deprecated method `SilverStripe\ORM\Connect\Query::seek()` - use `getIterator()` instead
+- Removed deprecated method `SilverStripe\ORM\Connect\Query::valid()` without equivalent functionality to replace it
+- Removed deprecated method `SilverStripe\ORM\DatabaseAdmin::updateLegacyClassNames()` - use `updateLegacyClassNameField()` instead
+- Removed deprecated method `SilverStripe\ORM\DataExtension::unload_extra_statics()` without equivalent functionality to replace it
+- Removed deprecated method `SilverStripe\ORM\DataList::getGenerator()` without equivalent functionality to replace it
+- Removed deprecated method `SilverStripe\ORM\DataObject::doValidate()` - use `validate()` instead
+- Removed deprecated method `SilverStripe\ORM\DataObject::duplicateManyManyRelations()` - use `duplicateRelations()` instead
+- Removed deprecated method `SilverStripe\ORM\DB::getConn()` - use `DB::get_conn()` instead
+- Removed deprecated method `SilverStripe\ORM\FieldType\DBClassName::clear_classname_cache()` - use `DBEnum::flushCache()` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:1061]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 22]
```diff
@@ -900,228 +907,231 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
-- Removed deprecated method `SilverStripe\Dev\SapphireTest::loadFixture()`
-- Removed deprecated method `SilverStripe\Dev\SapphireTest::setFixtureFactory()`
-- Removed deprecated method `SilverStripe\Dev\TestKernel::getIgnoredCIConfigs()`
-- Removed deprecated method `SilverStripe\Dev\TestKernel::setIgnoredCIConfigs()`
-- Removed deprecated method `SilverStripe\Dev\TestMailer::saveEmail()`
-- Removed deprecated method `SilverStripe\Forms\FieldList::collateDataFields()`
-- Removed deprecated method `SilverStripe\Forms\Form::formHtmlContent()`
+- Removed deprecated class `SilverStripe\Control\Email\SwiftMailer` - use `symfony/mailer` instead
+- Removed deprecated class `SilverStripe\Control\Email\SwiftPlugin` - use `symfony/mailer` instead
+- Removed deprecated class `SilverStripe\Control\RequestProcessor` - use `HTTPMiddleware` directly instead.
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:930]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

File: en/04_Changelogs/5.0.0.md [hunk 23]
```diff
@@ -694,33 +701,33 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
...
+- Removed deprecated method `SilverStripe\Assets\Flysystem\FlysystemAssetStore::getFilesystemFor()` - use `applyToFileIDOnFilesystem()` instead
+- Removed deprecated method `SilverStripe\Assets\Flysystem\FlysystemAssetStore::getOriginalFilename()` - use `FileResolutionStrategy::parseFileID()` and `ParsedFileID::getFilename()` instead
+- Removed deprecated method `SilverStripe\Assets\Flysystem\FlysystemAssetStore::getStreamSHA1()` - use `FileHashingService::computeFromStream()` instead
+- Removed deprecated method `SilverStripe\Assets\Flysystem\FlysystemAssetStore::getVariant()` - use `FileResolutionStrategy::parseFileID()` and `ParsedFileID::getVariant()` instead
+- Removed deprecated method `SilverStripe\Assets\Flysystem\FlysystemAssetStore::moveBetweenFilesystems()` - use `moveBetweenFileStore()` instead
+- Removed deprecated method `SilverStripe\Assets\Flysystem\FlysystemAssetStore::parseFileID()` - use `FileResolutionStrategy::parseFileID()` instead
+- Removed deprecated method `SilverStripe\Assets\Flysystem\FlysystemAssetStore::removeVariant()` - use `FileResolutionStrategy::parseFileID()` and `ParsedFileID::setVariant()` instead
```
Comment (sabina-talipova) [inline en/04_Changelogs/5.0.0.md:727]: FQN like here
https://github.com/silverstripe/developer-docs/blob/91551914320c698e8028d1257ef08bbae3f7f329/en/04_Changelogs/5.0.0.md?plain=1#L950

**Review by sabina-talipova — CHANGES_REQUESTED** [review]
I believe that we should specify the fully qualified class name so that it does not raise questions about which particular class we mean, especially if they are in another module or just has another namespace. 
If you think that this should be put in a separate ticket, I will be happy to create a new task.

**Comment by GuySartorelli** [comment]
> If you think that this should be put in a separate ticket, I will be happy to create a new task.

Please do raise that as a new card. This was just to pull the information that's in the `@deprecated` annotations in PHPDocs through to the changelog. I agree that it would be good for the changelog to provide even clearer information than what's here, but that's not in scope for the current issue and could be more effort than it's worth in the end. We'll need to take that through backlog refinement.

**Review by sabina-talipova — APPROVED** [review]
Looks good to me :)
Required changes will be implemented as part of another [ticket](https://github.com/silverstripe/developer-docs/issues/253)

**Pull request merged**